LetsDinnerV2 MessagesExtension/View/Cells/DescriptionCell/DescriptionCell.swift

```diff
@@ -19,6 +19,8 @@ class DescriptionCell: UITableViewCell, UICollectionViewDelegate, UICollectionVi
     private let selectedCustomRecipes = Event.shared.selectedCustomRecipes
     private var allRecipesTitles = [String]()
     
+    private let collectionViewMinimumLineSpacing = CGFloat(20)
```

Comment:
name is way too long, minLineSpacing is more than enough. Why CGFloat(20) and not : CGFloat = 20?